feg/gateway/services/s8_proxy/client_api.go

```diff
@@ -17,9 +17,13 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"strings"
+
+	"google.golang.org/grpc"
```

Comment:
I think the grpc include should be below glog

Reply:
ops. Thanks!